app/scripts/modules/amazon/loadBalancer/configure/createLoadBalancer.html

```diff
@@ -20,5 +20,6 @@
           ng-click="ctrl.cancel()">Cancel
   </button>
   <submit-button is-disabled="form.$invalid || taskMonitor.submitting"
-                 submitting="taskMonitor.submitting" on-click="ctrl.submit()" is-new="isNew"></submit-button>
+                 submitting="taskMonitor.submitting" on-click="ctrl.submit()"
+                 is-new="isNew" is-add="isAdd"></submit-button>
```

Comment:
It'd be cleaner to explicitly pass in the `label` attribute by setting it in the controller. This would follow the pattern in the server group wizards.